docs/en/testing/supplemental.md

```diff
@@ -243,56 +389,6 @@ v('div', {
 ])
 ```
 
-### `harness.expectPartial`
```

Comment:
Removed this because we don't really want to promote its use. With assertion templates it has become less useful as the templates can be easily manipulated but still test the whole widget.